STANDARDS.md

```diff
@@ -1,16 +1,66 @@
-# Dataset Standards
 
-This file outlines the standards and requirements needed for submitting a dataset to QCArchive.
+This file outlines the standards and data requirements needed for submitting a dataset to QCArchive.
 This ensures that we have a consistent data model for downstream processes.
 
-## Required fields 
+STANDARDS version: 3-DRAFT (2020-11-13)
+
+We distinguish between standards for the datasets (i.e. the actual data), and the standards for training force fields.
 
-Current list:
-* Ensure all submissions have cmiles, most important are mapped hydrogen smiles
-* Ensure the WBO is requested for all submissions, this should be included in the scf properties list using the flag `wiberg_lowdin_indices`
+# Dataset Standards
 
-## Best practices
-* If any calculations are to be redone from another collection, re-use the old input (coordinates, atom ordering etc) as this will avoid running the calculation again and will just create new references in the database to the old results and should help keep the cost of the calculations down.  
+## Each molecule must have the following information:
+- Canonical isomeric explict hydrogen mapped SMILES
+- Provenence info of SMILES generation (NEW)
+- Coordinates
+- Total Charge
+- Coordinates must be in CMILES order (discuss)
+
+## Each dataset must have the following information:
+- Name
+- Version (NEW)
+- A short description
+- A long description
+- A link/URL/reference pointing to the provenence to reproduce dataset (e.g. the GH submission folder)
+- A changelog (NEW)
+- A github submitter username
+- The name of the person who selected/sourced the molecules (NEW)
+- A description of the meaning of the entry/molecule keys/names (NEW)
+- Each entry has canonical isomeric explict hydrogen mapped SMILES
+- Provenence info of CMILES generation (NEW)
```

Comment:
This would be the same information as the per molecule smiles provenance info.

Reply:
Well, not necessarily given the 3D coordinates, but, this again asks whether CMILES info should be mutable QCA

Reply:
Could be a precendence model, where top-level dataset metadata applies to all molecules *unless* that molecule has its own, perhaps conflicting metadata; finest scope (molecule-specific) wins where present.

Reply:
I agree with this, but there are complications that have prevented this. For example, QCA does not have the notion of data precedence, which is why we cannot for example take CMILES info from an entry and instead need to store it in the molecule. If there was a precedence model in place, then the molecule wouldn't need CMILES info in the first place.

Reply:
In the name of getting this document through, I will hold off on this since it would require extra implementation work and has less value per person-hour than other issues. I am a big of this working, so I am hopeful that it gets adopted in later versions.